maps/src/main/java/org/odk/collect/maps/layers/OfflineMapLayersImporter.kt

```diff
@@ -21,19 +21,21 @@ class OfflineMapLayersImporter(
     val viewModel: OfflineMapLayersViewModel by activityViewModels {
         object : ViewModelProvider.Factory {
             override fun <T : ViewModel> create(modelClass: Class<T>): T {
-                return OfflineMapLayersViewModel(referenceLayerRepository, scheduler, settingsProvider) as T
+                return OfflineMapLayersViewModel(
+                    referenceLayerRepository,
+                    scheduler,
+                    settingsProvider
+                ) as T
             }
         }
     }
 
-    private lateinit var binding: OfflineMapLayersImporterBinding
-
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-        binding = OfflineMapLayersImporterBinding.inflate(inflater)
+        val binding = OfflineMapLayersImporterBinding.inflate(inflater)
```

Comment:
This is something I always forget about! It's dangerous to hold bindings in a field in `Fragment`: https://developer.android.com/topic/libraries/view-binding#fragments.